datasetviewer/preview/PreviewPresenter.py

```diff
@@ -0,0 +1,47 @@
+from datasetviewer.presenter.SubPresenter import SubPresenter
+from datasetviewer.mainview.interfaces.MainViewPresenterInterface import MainViewPresenterInterface
+
+
+class PreviewPresenter(SubPresenter):
+
```

Comment:
Need for code comments already discussed. 

Can you eliminate empty lines in the code you have written.